tests/transform.spec.ts

```diff
@@ -121,6 +125,25 @@ describe('transform tests', () => {
         content: 'import("@/components/test").Test;',
         output: "import('../components/test').Test;"
       },
+      {
+        // https://github.com/qmhc/vite-plugin-dts/issues/330
+        description: 'wildcard alias at root level with relative import',
+        filePath: './src/components/Sample/index.ts',
+        content: 'import { Sample } from "./Sample";',
+        output: "import { Sample } from './Sample';\n"
+      },
```

Comment:
This is the added test case that was failing initially. The path was incorrectly being adjusted to `../../Sample` because the path replacement pattern `/^(.+)/` generated by the `'*'` path alias was too eagerly matching the relative path and making it absolute.